src/Replicator.js

```diff
@@ -180,7 +180,8 @@ class Replicator extends EventEmitter {
     this.emit('load.progress', this._id, hash, latest, null, this._buffer.length)
 
     // Return all next pointers
-    return log.values.map(getNext).reduce(flatMap, [])
+    const refsPointers = log.values.map(nextRefsUnion).reduce(flatMap, [])
+    return refsPointers
```

Comment:
probably best to only change the name of the function given to .map; no unnecessary variable declaration and less commit diff.